apps/site/pages/docs/api-extensions/extending-api-schema.mdx

```diff
@@ -0,0 +1,378 @@
+ 
+### Ensure your development server is active
+ 
+Ensure that your development server is up and running while developing to see the changes you make. You can do this by running the command `yarn dev` in your terminal, which will monitor the changes.
+
+</Steps>
+
+---
+
+## Extending VTEX API Schemas 
+
+FastStore streamlines the way of using data from VTEX APIs that the FastStore API does not expose.
+
+As FastStore uses GraphQL, it's required to write [type definitions](https://graphql.org/learn/schema/#object-types-and-fields) and [resolvers](https://graphql.org/learn/execution/#root-fields-resolvers) to fetch the data you need. 
+To define new type definitions and resolvers for FastStore using GraphQL, learn how to create a specific directory structure in the following step by step. 
+
+---
+
+### Step by step
+#### Step 1 - Preparing the folders and files 
+
+To declare a new type definition and resolver you need to create a directory structure:
+
+1. In you store repository, go to the `src` folder and create the `graphql` subfolder. You can do this by running the following command in your terminal:
+
+<Tabs items={['macOs and Linux', 'Windows']}>
+  <Tab>
+  ```bash copy
+  mkdir graphql
+  ```
+  </Tab>
+
+  <Tab>
+   ```bash copy
+  mkdir graphql
+  ```
+  </Tab>
+</Tabs>
+
+2. Inside the new `graphql` folder, create the `vtex` subfolder.
+
+<Tabs items={['macOs and Linux', 'Windows']}>
+  <Tab>
+  ```bash copy
+  mkdir vtex
+  ```
+  </Tab>
+
+  <Tab>
+   ```bash copy
+  mkdir vtex
+  ```
+  </Tab>
+</Tabs>
+
+3. In `vtex`, create `resolvers` and `typeDefs` subfolders.
+<Tabs items={['macOs and Linux', 'Windows']}>
+  <Tab>
+  ```bash copy
+  mkdir resolvers/typeDefs
+  ```
+  </Tab>
+
+  <Tab>
+   ```bash copy
+  mkdir resolvers\typeDefs
+  ```
+  </Tab>
+</Tabs>
+
+<Callout type="warning" emoji="⚠️">
+  The `src/graphql/vtex/resolvers` and `src/graphql/vtex/typeDefs` path is mandatory, but you can organize the files inside those directory as you wish.
+</Callout>
+
+4. Create a `index.ts` file inside the `resolvers` folder.
+
+Once you have created the essentials folders, you will have a folder structure for the VTEX API Schema Extensions in the `src/graphql/vtex` folder similiar to the the following:
+
+```ts copy
+starter.store/
+└─ src/
+    └─ graphql/
+        └─ vtex/
+          ├─ resolvers/
+              └─ index.ts
+              └─ <resolverName>.ts
+              └─ …
+          ├─ typeDefs/
+              └─<typeName>.graphql
+              └─ …
+```
+
+<Callout type="info" emoji="ℹ️">
+  Note that in the code example above, the type definitions and resolvers file were are created ad `<resolverName>.ts` and `<typeName>.graphql`.
+</Callout>
+
+
+#### Step 2 - Creating type definitions (`TypeDefs`)
```

Comment:
```suggestion
#### Step 2 - Creating type definitions (`typeDefs`)
```

I'd change casing to avoid confusion. Using this casing in the folder name makes it don't work.